app-finatra-test/src/test/scala/net/spals/appbuilder/app/finatra/sample/SampleFinatraWebApp.scala

```diff
@@ -0,0 +1,34 @@
+package net.spals.appbuilder.app.finatra.sample
+
+import net.spals.appbuilder.app.finatra.FinatraWebApp
+import net.spals.appbuilder.config.service.ServiceScan
+import net.spals.appbuilder.executor.core.ManagedExecutorServiceRegistry
+import net.spals.appbuilder.filestore.core.FileStore
+import net.spals.appbuilder.mapstore.core.MapStore
+import net.spals.appbuilder.message.core.{MessageConsumer, MessageProducer}
+import net.spals.appbuilder.model.core.ModelSerializer
+
+/**
+  * A full sample [[FinatraWebApp]] which uses all default services
+  * and bindings.
+  *
+  * @author tkral
+  */
+class SampleFinatraWebApp extends FinatraWebApp {
+
+  setServiceConfigFromClasspath("config/sample-finatra-service.conf")
+  setServiceScan(new ServiceScan.Builder()
+    .addServicePackages("net.spals.appbuilder.app.finatra.sample")
+    .addDefaultServices(classOf[ManagedExecutorServiceRegistry])
+    .addDefaultServices(classOf[FileStore])
+    .addDefaultServices(classOf[MapStore])
+    .addDefaultServices(classOf[MessageConsumer], classOf[MessageProducer])
+    .addDefaultServices(classOf[ModelSerializer])
+    .build())
+  addModule(new SampleGuiceModule)
+  addModule(new SampleTwitterModule)
+  build()
+
+}
```

Comment:
Sample application which uses the `appbuilder` framework with Finatra support. This is only default services (i.e. no plugins).